config/config.go

```diff
@@ -157,6 +158,8 @@ type TracingConfig struct {
 	Enabled   bool   `yaml:"enabled"`
 	Namespace string `yaml:"namespace"`
 	Service   string `yaml:"service"`
+	Port      int32  `yaml:"port"`
```

Comment:
That must be added in the CR files too (as doc in `kiali_cr.yaml`, and `main.yml`)

Reply:
Good point